pkg/restapi/issuer/operation/operations.go

```diff
@@ -383,6 +384,18 @@ func (o *Operation) createIssuerProfileHandler(rw http.ResponseWriter, req *http
 		return
 	}
 
+	if profileData.SupportsWACI {
+		for _, pCredScope := range profileData.CredentialScopes {
+			if _, ok := o.cmOutputDescriptor[pCredScope]; !ok {
+				commhttp.WriteErrorResponseWithLog(rw, http.StatusInternalServerError,
+					fmt.Sprintf("failed to get output descriptors configured for waci "+
```

Comment:
Two things:

1. This string has no formatting directives, so you can simply use the string without calling `fmt.Sprintf`
2. I think this error message isn't specific enough - why couldn't the output descriptors get configured? What exactly went wrong?

Reply:
It has formatting directives now